kornia/augmentation/_2d/intensity/median_blur.py

```diff
@@ -0,0 +1,51 @@
+from typing import Any, Dict, Optional, Tuple
+
+from torch import Tensor
+
+from kornia.augmentation._2d.intensity.base import IntensityAugmentationBase2D
+from kornia.filters import median_blur
+
+
+class RandomMedianBlur(IntensityAugmentationBase2D):
+    """Add random blur with a median filter to an image tensor.
+
+    .. image:: _static/img/RandomMedianBlur.png
```

Comment:
You need to update this file
https://github.com/kornia/kornia/blob/master/docs/generate_example_images.py